sklearn/metrics/tests/test_common.py

```diff
@@ -595,7 +596,7 @@ def test_invariance_string_vs_numbers_labels():
 
     for name, metric in THRESHOLDED_METRICS.items():
         if name in ("log_loss", "hinge_loss", "unnormalized_log_loss",
-                    "brier_score_loss"):
+                    "brier_score_loss", "roc_auc_score"):
             # Ugly, but handle case with a pos_label and label
```

Comment:
not sure what "and label" means here. Any idea?

What metrics are we excluding here? Why do we need an explicit list of names where otherwise these lists are defined at the top with meaningful names?

I know it's not your problem, but this is a mess :\